README.md

```diff
@@ -49,14 +49,14 @@ jobs:
       - name: Check for code changes to the action
         id: action-code
         # You may also reference just the major or major.minor version
-        uses: im-open/did-custom-action-code-change@v1.0.2
+        uses: im-open/did-custom-action-code-change@v1.0.3
         with:
           files-with-code: 'action.yml,package.json,package-lock.json'
           folders-with-code: 'src,dist'
           token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Commit unstaged changes if there are code changes
-        if: steps.action-code.outputs.HAS_CHANGES == 'true'
+        if: steps.action-code.outputs.HAS_CHANGES == 'true' # can also use env.HAS_CHANGES
```

Comment:
Are we worried at all that `HAS_CHANGES` seems like a potentially common variable name and might conflict with workflows this is used in?

Reply:
I was literally just going to fix this. :)  In my other action where I made a similar PR I changed it to FILE_HAS_CHANGED.  So I'll do something similar here.

Reply:
This one has bee renamed to CODE_HAS_CHANGED.  I don't like that the output and env don't match but changing the output isn't realistic at this point.

Reply:
We could update the output and make a major version update 😃 But that might be too much. I just 100% agree that having the output and env names be different sucks.